docs/installation.rst

```diff
@@ -121,3 +121,71 @@ email address. Then we could do this:
 Then you'd use the Python dotted path to that class in the
 ``settings.AUTHENTICATION_BACKENDS`` instead of
 ``mozilla_django_oidc.auth.OIDCAuthenticationBackend``.
+
+
+Creating a Django ``User`` record for new users
+-----------------------------------------------
+
+If a user logs into your site and doesn't already have an account, by default,
```

Comment:
nit: Shall we also mention something about the `OIDC_CREATE_USER`?  Eg that the default behavior can be changed by the said variable or does it feel too verbose?